integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java

```diff
@@ -202,6 +210,16 @@ public EnvironmentTestDto withS3Guard() {
         return this;
     }
 
+    public EnvironmentTestDto withParentEnvironmentName(String parentEnvironmentName) {
```

Comment:
it would make sense to implement this method with RunninParameter argument and let the method gain the environment name from the testcontext
withParentEnvironmentName(RunningParameter parentEnvKey) {
  getRequest().setEnvironmentName(resource.get(parentEnvKey.getKey());
...

but it is just nice to have